src/demo/share/jfc/Font2DTest/Font2DTest.java

```diff
@@ -268,11 +268,6 @@ private void setupMenu( boolean isApplet ) {
         fileMenu.add( new MenuItemV2( "Page Setup...", this ));
         fileMenu.add( new MenuItemV2( "Print...", this ));
         fileMenu.addSeparator();
-        if ( !isApplet )
-          fileMenu.add( new MenuItemV2( "Exit", this ));
```

Comment:
If it is not Applet, should we not retain this line?

Reply:
oopsy. I did not mean to do that. I will restore.